<?xml version="1.0" encoding="utf-8"?>
<launch>

  <!-- these are the arguments you can pass this launch file, for example paused:=true -->
  <arg name="paused" default="false"/>
  <arg name="use_sim_time" default="true"/>
  <arg name="gui" default="true"/>
  <arg name="headless" default="false"/>
  <arg name="debug" default="false"/>
  <arg name="config" default="true"/>


  <node pkg="tf" type="static_transform_publisher" name="kinect_transform" args="0.095 0 0.815 0 1 0 0 base openni_depth_optical_frame 100" />

  <include file="$(find baxter_moveit_config)/launch/planning_context.launch">
    <arg name="load_robot_description" value="true"/>
  </include>


  <include file="$(find baxter_moveit_config)/launch/move_group.launch">
    <arg name="allow_trajectory_execution" value="true"/>
    <rosparam command="load" file="$(find me_495_final)/config/kinematics.yaml"/>
	<rosparam name ="move_group/trajectory_execution/allowed_start_tolerance" type="double" value="1" />
  </include>

  <node pkg="me_495_final" type="move_arm.py" name="move_arm" output="screen">
    <remap from="/joint_state" to="/robot/joint_states"/>
  </node>

</launch>
